NDcPP_v2_2e.adoc

```diff
@@ -2835,19 +2831,19 @@ _List 2: List of supported TLS-related ciphersuites for TLSv1.3_
 
 *FCS_TLSC_EXT.1 TLS Client Protocol*
 
-*FCS_TLSC_EXT.1.1* The TSF shall implement [selection: _TLS 1.3 (RFC 8446), TLS 1.2 (RFC 5246)_] and [selection:  _TLS 1.1 (RFC 4346), no additional TLS version_] and reject all other TLS and SSL versions. The TLS implementation will support the following ciphersuites:
+*FCS_TLSC_EXT.1.1* The TSF shall implement [selection: _TLS 1.3 (RFC 8446), TLS 1.2 (RFC 5246)_] and [assignment:  _no additional TLS version_] and reject all other TLS and SSL versions. The TLS implementation will support the following ciphersuites:
 
 [selection:
 
-* [.underline]#_select supported ciphersuites for TLSv1.1 and/or TLSv1.2 from List 1_#
+* [.underline]#_select supported ciphersuites for TLSv1.2 from List 1_#
 
 * [.underline]#_select supported ciphersuites for TLSv1.3 from List 2_#
 
 ] _and no other ciphersuites_.
 
 *_Application Note {counter:appnote_count}_*
 
-_The ciphersuites to be tested in the evaluated configuration are limited by this requirement and must be selected from the ciphersuites defined in List 1 (TLSv1.1/v1.2) and List 2 (TLSv1.3), respectively. The ST author should select the ciphersuites that are supported. Even though RFC 5246 mandates implementation of specific ciphers for TLSv1.2 this cPP depreciates use of SHA-1. Support for TLS_RSA_WITH_AES_128_CBC_SHA is not claimed in order to claim conformance to this cPP._
+_The ciphersuites to be tested in the evaluated configuration are limited by this requirement and must be selected from the ciphersuites defined in List 1 (TLSv1.2) and List 2 (TLSv1.3), respectively. The ST author should select the ciphersuites that are supported. Even though RFC 5246 mandates implementation of specific ciphers for TLSv1.2 this cPP depreciates use of SHA-1. Support for TLS_RSA_WITH_AES_128_CBC_SHA is not claimed in order to claim conformance to this cPP._
```

Comment:
Nitpick: The term "depreciate" (lower value) doesn't really apply here.  I think the correct term is "deprecates" (to phase out).